.github/actions/build-frontend/action.yml

```diff
@@ -18,19 +18,24 @@ runs:
     - name: Build release for production branch (conditional check)
       if: github.ref == 'refs/heads/production'
       working-directory: frontend-react
-      run: yarn build:production
+      run: |
+        echo "APPINSIGHTS_KEY=${{secrets.APPINSIGHTS_PROD_KEY}}" >> .env.production
```

Comment:
Can we make it `REACT_APP_APPINSIGHTS_KEY` in both places? React restricts what variables get exposed to the runtime.